framework_tvb/tvb/interfaces/web/controllers/burst/matjax.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+#
+#
+# TheVirtualBrain-Framework Package. This package holds all Data Management, and
+# Web-UI helpful to run brain-simulations. To use it, you also need do download
+# TheVirtualBrain-Scientific Package (for simulators). See content of the
+# documentation-folder for more details. See also http://www.thevirtualbrain.org
+#
+# (c) 2012-2020, Baycrest Centre for Geriatric Care ("Baycrest") and others
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software Foundation,
+# either version 3 of the License, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License along with this
+# program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+#   CITATION:
+# When using The Virtual Brain for scientific publications, please cite it as follows:
+#
+#   Paula Sanz Leon, Stuart A. Knock, M. Marmaduke Woodman, Lia Domide,
+#   Jochen Mersmann, Anthony R. McIntosh, Viktor Jirsa (2013)
+#       The Virtual Brain: a simulator of primate brain network dynamics.
+#   Frontiers in Neuroinformatics (7:10. doi: 10.3389/fninf.2013.00010)
+#
+#
+
+
+def configure_matjax_doc(available_models):
+    models_docs = []
+
+    for clz_name, clz in available_models.items():
+        models_docs.append({
+            'name': clz_name.replace(' ', '_'),
+            'inline_description': _dfun_math_directives_to_matjax(clz),
+            'description': clz.__doc__
+        })
+
+    return models_docs
+
+
+def _dfun_math_directives_to_matjax(model):
+    """
+    Looks for sphinx math directives if the docstring of the dfun function of a model.
+    It converts them in html text that will be interpreted by mathjax
+    The parsing is simplistic, not a full rst parser.
+    """
+    def format_doc(doc):
+        return multiline_math_directives_to_matjax(doc).replace('&', '&amp;').replace('.. math::', '')
+
+    try:
+        doc = model.dfun.__doc__
+    except AttributeError:
+        doc = None
+
+    if doc is not None:
+        return format_doc(doc)
+
+    # try the parent __doc__
+    try:
+        doc = model.__mro__[1].dfun.__doc__
+    except (AttributeError, IndexError):
+        doc = None
+
+    if doc is not None:
+        return format_doc('Documentation is missing. Copy-ed from parent\n' + doc)
+
+    return 'Documentation is missing. '
+
+
+def multiline_math_directives_to_matjax(doc):
```

Comment:
This function could be renamed `_ multiline_math_directives_to_matjax ` for making it explicit it is not part of the public API of the current module?
Or is it used somewhere?

Reply:
No, it is not.